packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -0,0 +1,261 @@
+  getSansBold,
+  getSansRegular,
+  getSerifMedium,
+} from '@bbc/psammead-styles/font-styles';
+import {
+  getLongPrimer,
+  getMinion,
+  getPica,
+} from '@bbc/gel-foundations/typography';
+import { Link } from '@bbc/psammead-story-promo';
+import { oneOf, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+const CardWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+  background-color: ${C_WHITE};
+  position: relative;
+  border: 0.0625rem solid transparent;
+`;
+
+const TextWrapper = styled.div`
+  padding: 0 ${GEL_SPACING};
+`;
+
+const HeadingWrapper = styled.h3`
+  ${({ service }) => service && getSerifMedium(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerTextColor }) => headerTextColor};
+  margin: 0; /* Reset */
+`;
+
+const HeadingContentWrapper = styled.span.attrs({ role: 'text' })``;
+
+const LabelWrapper = styled.span`
+  ${({ service }) => service && getSansBold(service)};
+  ${({ script }) => script && getPica(script)};
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const TitleWrapper = styled.span`
+  color: ${({ titleColor }) => titleColor};
+  padding: ${GEL_SPACING} 0;
+  display: block;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled.p`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getLongPrimer(script)};
+  color: ${C_METAL};
+  padding-bottom: ${GEL_SPACING_DBL};
+  margin: 0; /* Reset */
+`;
+
+const ButtonWrapper = styled.div`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getMinion(script)};
+  padding: ${GEL_SPACING};
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  color: ${({ durationColor }) => durationColor};
+  border-top: 1px solid transparent;
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${({ durationColor }) => durationColor};
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.time`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const DurationTextWrapper = styled.span.attrs({ 'aria-hidden': 'true' })``;
+
+const programStateConfig = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    titleColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+};
+
+const sentenceCase = text =>
```

Comment:
Is this just used to capitalise the duration label? Would we be able to just pass in the translation string like this, we do that elsewhere :)

Reply:
Ah it's also used below for the live label text. We have to do exactly the same in the TV and Radio bulletin I believe? Can we follow the same approach in both places?

Reply:
We're using this for the visually-hidden state label as well,

Reply:
We're using this on the **state label** as well as the **duration label**. If the state is live, we need to display localised  `LIVE` and have `Live` as a hidden text on the card. The Radio bulletin [has a hardcoded hidden-text `Live`](https://github.com/bbc/simorgh/blob/2b82268bbf23ebf961cdc31e47b59f2068b93647/src/app/containers/Bulletin/index.jsx#L89) regardless of the service.

Reply:
I think the requirement _should_ be the same for both - if we're hardcoding Live for all cases on the Bulletin and have the requirement to only do that if the translation is 'Live' for the radio schedule, we should check that this should be the case, I imagine they should be consistent. At the moment it looks like we're always displaying 'LIVE' as the visible text for the bulletin component too.

Let's check the following for bulletin and radio schedules are true:

Scenario 1 (LTR services):
- Show a visible 'LIVE' label in english
- Show visually hidden text with 'Live' read in an english accent

Scenario 2 (RTL services):
- Show a visible live label, in capitals, in the service language.
- Show visually hidden text with the translation of live read in the service language.

This is what I understood, but as you point out, currently in Simorgh we never pass a live label translation for bulletins even in RTL languages, so it defaults to 'LIVE', and the visually hidden text is hardcoded as 'Live'.

Reply:
Stupid question, but do our right to left scripts contain capital letters? I don't think arabic does?

Reply:
Another question, can we pass the translation of 'Live' in all cases in this format, and use `text-transform: uppercase` in the styles for the visible live label?

Reply:
"Show visually hidden text with the translation of live read in the service language." - this is not required for services that do not have 'live' in english

Reply:
Just had a conversation with Catharine:
- We only need visually hidden text for ‘Live’ in English as a hack to stop screen readers reading out ‘live’ incorrectly, we don’t need it for other languages.
- We _don’t_ want to read out ‘Live’ in English when we are using a different language, so it looks like the bulletin component is currently incorrect. We have no RTL languages using this component yet, so this won’t have impacted anyone.
- We can pass in translation strings as ‘Live’ and ‘Duration’, not need the sentenceCase function, and use CSS (or toUpperCase as you are) to display the make it ‘LIVE’.

Reply:
> Another question, can we pass the translation of 'Live' in all cases in this format, and use `text-transform: uppercase` in the styles for the visible live label?

This will cause UX and accessibility issues in `no-CSS `browsers

Reply:
I'm gonna move that function to a helper and use it in `tests` and `stories`

Reply:
Could you not just pass the translations strings in sentence case? You could still use toUpperCase for the visible label